tools/symlink.js

```diff
@@ -0,0 +1,70 @@
+const fs = require("fs-extra");
+const path = require("path");
+const yargs = require("yargs");
+const { hideBin } = require("yargs/helpers");
+
+/**
+ * @typedef {{dataPath: string;symLinkName: string;}} FoundryConfig
+ */
+
+const argv = yargs(hideBin(process.argv))
+  .option("clean", {
+    alias: "c",
+    type: "boolean",
+    default: false,
+  })
+  .parseSync();
+
+/**
+ * @type {FoundryConfig}
+ */
+const foundryConfig = fs.readJSONSync(
+  path.resolve(__dirname, "..", "foundryconfig.json")
+);
+
+const linkDirectory = path.resolve(
+  foundryConfig.dataPath,
+  "Data",
+  "systems",
+  foundryConfig.symLinkName
+);
+
+if (argv.clean) {
+  handleProcess(removeSymlink());
+} else {
+  handleProcess(createSymlink());
+}
+
+async function createSymlink() {
```

Comment:
This is async, but not awaiting anything. Should there be an await here?

Reply:
possibly a question for me, since this is mainly a copy/paste from the gulpfile. Checking what happened here now

Reply:
I made it async as the `handleProcess` function expects the pass parameter to be a Promise (the call to `.then`).

At the very least, I'll rename `handleProcess` to `handlePromise`. I'm pretty sure that's what I _wanted_ to name it in the first place, but mistyped.

Reply:
I could also return `Promise.resolve` or `Promise.reject`, but making it `async` was more straightforward in my mind.

Reply:
I'm realizing now, that I can just use the `*Sync` functions across the board in this file. I'll do that and remove the need for `handleProcess`.